tf2_ros_py/test/test_buffer.py

```diff
@@ -56,15 +55,15 @@ def test_can_transform_valid_transform(self):
         rclpy_time = clock.now()
         transform = self.build_transform('foo', 'bar', rclpy_time)
 
-        self.assertEqual(buffer.set_transform(transform, 'unittest'), None)
-
-        self.assertEqual(buffer.can_transform('foo', 'bar', rclpy_time), True)
+        assert buffer.set_transform(transform, 'unittest') == None
```

Comment:
I believe the general guidance is python is to use `is None`, to avoid possible custom comparison operators:

```suggestion
        assert buffer.set_transform(transform, 'unittest') is None
```